src/config.ts

```diff
@@ -16,5 +16,13 @@ export const RETRY_INTERVAL = 10000;
 export const POLLING_INTERVAL = 30000;
 export const CONFIRMATIONS = 1;
 export const CONFIRMATION_TIMEOUT = 10 * 60 * 1000; // 10 minutes
-export const GAS_MULTIPLIER = 160;
+export const GAS_LIMIT_MULTIPLIER = 160;
+export const GAS_PRICE_MULTIPLIER = 160;
 export const BALANCE_THRESHOLD = parseEther("0.05");
+export const GAS_STATION_API_ENABLED = true;
```

Comment:
I'm not sure if I'd enabled it by default. Do you mind leaving it as false?

Reply:
done in 0f2b66e